### PR TITLE
show age and uncertainty labels and event settings

### PR DESCRIPTION
**Pull request by paologum**
image

image

image

- Event Specific settings that deal with `type` and `rangeSort` are implemented
- fixed columns tab not displaying correctly on a resized window
- show age and uncertainty labels
- class name and style cleanup
- changed timetree datapack to vertebrate because timetree mega tree takes too much time and memory for it to be usable in current dev state
- (vertebrate and hominids have a column with the same name so it creates a small bug that is posted on the Jira already)
- new general radio component
- transitioning towards lodash deep clone library because `JSON.stringify` cloning doesn't preserve type and has problems with objects or types that are not primitive (undefined as well)

### CODE REVIEW

File: shared/src/index.ts [hunk 1]
```diff
@@ -199,6 +201,8 @@ export type SubInfo =
   | SubSequenceInfo
   | SubTransectInfo;
 
+export type ColumnSpecificSettings = EventSettings;
```
[the remark this answers is not in the document]
Comment (paologum) [reply shared/src/index.ts:204]: this is for the event specific settings, and will be a union for any more columns that will need specific settings for their type

**Review by jay-jhk-lee — APPROVED** [review]
looks good

**Comment by asivath** [comment]
are the radio buttons going off the the blue area?

**Comment by paologum** [comment]
yeah they are, but it's because the container of the settings menu isn't styled. it's already a task i've put in the jira to reformat the whole menu so i'm leaving it for that task since it's probably gonna get overhauled

**Review by asivath — APPROVED** [review]

**Pull request merged**